shoes-core/spec/shoes/shared_examples/clickable.rb

```diff
@@ -2,4 +2,18 @@
   it { is_expected.to respond_to :click }
   it { is_expected.to respond_to :release }
   it { is_expected.to respond_to :register_click }
+
+  it "can chain click" do
+    result = subject.click do
+    end
```

Comment:
I always find the `{}` makes me think it's a hash... will see if Rubocop is happy with `do end` since I also wasn't a huge fan of the separate lines.